toolbox/core/bst_get.m

```diff
@@ -1180,10 +1180,10 @@
             return;
         end
         % Get list of current protocol subjects
-        ProtocolSubjects = GlobalData.DataBase.ProtocolSubjects(GlobalData.DataBase.iProtocol);
+        ProtocolSubjects = bst_get('ProtocolSubjects');

```

Comment:
This is not an acceptable solution... we have SQL now we should make use of it to make this much more efficient.

Reply:
1. Get the Subject ID from the AnatomyFile table using the FileName: sFile = db_get('AnatomyFile', SurfaceFile, 'Subject')
2. sSubject = bst_get('Subject', sFile.Subject);
3. iSubject = sSubject.Id
4. iSurface = find(file_compare(SurfaceFile, {sSubject.Surface.FileName}), 1);

Reply:
Solved in 3834c6560c3d94138f25784b7f976ee4029b297a
While possible, bst_get('SurfaceFile') calls will be replaced by db_get('AnatomyFile')